src/Nethermind/Nethermind.Merge.Plugin/MergeSealValidator.cs

```diff
@@ -32,9 +32,9 @@ public class MergeSealValidator : ISealValidator
         _poSSwitcher = poSSwitcher;
         _preMergeSealValidator = preMergeSealValidator;
     }
-    public bool ValidateParams(BlockHeader parent, BlockHeader header) => 
+    public bool ValidateParams(BlockHeader parent, BlockHeader header) =>
         _poSSwitcher.IsPostMerge(header) || _preMergeSealValidator.ValidateParams(parent, header);
 
-    public bool ValidateSeal(BlockHeader header, bool force) => 
-        _poSSwitcher.IsPostMerge(header) || _preMergeSealValidator.ValidateSeal(header, force);
+    public bool ValidateSeal(BlockHeader header, bool force) =>
+        _poSSwitcher.GetBlockConsensusInfo(header, true).IsPostMerge || _preMergeSealValidator.ValidateSeal(header, force);
```

Comment:
qq: why we don't check the seal if it is post merge?

Reply:
It is PoW specific thing, so we don't have it post merge

Reply:
Isnt seal mean the hash?

Reply:
Isnt seal mean the hash?

Reply:
https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.Consensus.Ethash/EthashSealValidator.cs#L77